s2/s2_test.go

```diff
@@ -31,6 +31,11 @@ var (
 	// When using blaze/bazel add "--test_arg=--benchmark_brute_force=true"
 	benchmarkBruteForce = flag.Bool("benchmark_brute_force", false,
 		"When set, use brute force algorithms in benchmarking.")
+
+	// To set in go testing add "--s2_random_seed=123" to your test command.
+	// When using blaze/bazel add "--test_arg=--s2_random_seed=123"
+	s2RandomSeed = flag.Int64("s2_random_seed", 1,
```

Comment:
Having a fixed seed seems bad, since then we're always testing the same seed. Can we have a default that means "pick a random seed"?

Reply:
Yes, that sounds reasonable.